src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -129,6 +130,18 @@ public void clear() {
         internalList.clear();
     }
 
+    /**
+     * Sorts all persons in address book by name
+     * @return success/failure of operation
+     */
+    public boolean sort() {
+        if(internalList.isEmpty()){
```

Comment:
Such statement requires a whitespace after keywords like `if` `while`.
You may leave it as it is for now, however in 4kloc, such coding standards will be enforced, you might want to refer to the coding standards for the module.